src/tools/sdk/archive/index.md

```diff
@@ -21,27 +21,43 @@ Go to the [Dart SDK page](/tools/sdk).
   {% include_relative _sdk-terms.md %}
 </aside>
 
+
+## Stable channel
+
+{% if site.data.pkg-vers.SDK.channel == 'dev' %}
+The stable channel currently contains 1.x and earlier versions of Dart.
```

Comment:
Why the "and earlier"?

Reply:
There are some 0.* releases in the archive.

Reply:
Oh, ok I didn't know.